frontend/src/views/SQR/Form.vue

```diff
@@ -488,18 +488,15 @@ export default class Form extends Vue {
     this.debounceSubmitDraft.cancel();
   }
 
-  async submitForm(save = false) {
-    const sqrSubmissionBranch = save
+  async submitForm(submit = false) {
+    const sqrSubmissionBranch = submit
       ? SubmissionsBranch.COMPLETED
       : SubmissionsBranch.DRAFTS;
-    if (!save) {
-      this.draftStatus = FormState.SAVING;
+    if (!submit) {
+      this.formStateMessages[FormState.SAVING] = "Saving...";
+      this.formStatus = FormState.SAVING;
     }
-    if (!save || (this.$refs as any).form.validate()) {
-      if (save) {
-        this.cancelAutoSave();
-        this.draftStatus = FormState.SAVING;
-      }
+    if (!submit || (this.$refs as any).form.validate()) {
```

Comment:
This `if` is long, till the end of the method. Avoid such long `if`s, rather negate the condition and return here.

https://dzone.com/articles/code-smells-if-statements